assets/src/data/content/writers/writer.ts

```diff
@@ -20,6 +16,7 @@ export interface WriterImpl {
   h6: ElementWriter;
   img: ElementWriter;
   youtube: ElementWriter;
+  iframe: ElementWriter;
```

Comment:
iframes were not supported in the html renderer used by activities

Reply:
good catch